vite.config.ts

```diff
@@ -25,6 +31,9 @@ export const ViteConfig = {
     }),
     eslintPlugin({ fix: true, cache: true, failOnWarning: true }),
   ],
+  resolve: {
+    alias: nuxtAliases,
+  },
```

Comment:
We now need three copies of what's almost the same alias set? 😵

Reply:
Yes, vite.config and nuxt.config both use the same aliases but we can just import and be fine :)